lib/mavenlink/request.rb

```diff
@@ -194,8 +194,10 @@ def each_page(batch_size = 200, &block)
 
         while (records_passed += page_records.count) < total_count
           response = request.page(i+=1).perform
+          puts response.results
```

Comment:
Do we really want to print the results?

Reply:
Yeah, I think we should remove it

Reply:
🤦‍♂ Oops. Was having trouble with the specs. Removed.